app/views/design-system/rsd-design-system/design-patterns/contact-details/example-contact-per-row-read-only.html

```diff
@@ -0,0 +1,76 @@
+{% extends "./../example-layout.html" %}
+
+{% block pageTitle %}
+Contact details design pattern - contact per row (read only) | RSD component library
+{% endblock %}
+
+{% block sample %}
+<div class="govuk-summary-card">
+  <div class="govuk-summary-card__title-wrapper">
+    <h2 class="govuk-summary-card__title">
+      DfE contacts
+    </h2>
+  </div>
+  <div class="govuk-summary-card__content">
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Trust relationship manager 
+        </dt>
+        <dd class="govuk-summary-list__value">
+          Alex James<br/>
+          alex.james@education.gov.uk
+        </dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          SFSO (Schools financial support and oversight) lead
+        </dt>
+        <dd class="govuk-summary-list__value">
+          Zara Laney<br/>
+          zara.laney@education.gov.uk
```

Comment:
See previous comments about emails

Reply:
We have replaced all @education.gov.uk emails with @example.com